environment_unix.yml

```diff
@@ -40,7 +40,7 @@ dependencies:
   # RocksDB is not supported on the conda build at the moment
   #  - rocksdb
   # ArcticDB is currently incompatible with fmt 10
```

Comment:
What about this comment ` # ArcticDB is currently incompatible with fmt 10` is it compatible now, what changed?

Reply:
I can't really remember to be honest.

It might have been something related with folly when it was not pinned, or a missing header inclusion as fixed by dbe011d.

This comment has been removed by 3a05633.